ForEachGroupBy/groupby/action_groupby.py

```diff
@@ -0,0 +1,54 @@
+from sekoia_automation.action import Action
+
+
+class GroupProcessor(Action):
+    """
+    Action to group items in a list by a specified key and optionally filter them.
+    """
+
+    def run(self, arguments):
+        group_key = arguments.get("group_key", 0)  # key to group by
+        filter_key = arguments.get("filter_key", None)  # key to filter by, if provided
+        filter_value = arguments.get("filter_value", None)  # value to filter by, if provided
+        input_data = arguments.get("input", [])  # input array of elements
+
+        # Dictionary to hold grouped elements
+        grouped = {}
+
```

Comment:
Can you check in `input_data` is a list?

Reply:
Done